viam-cartographer/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ endif()
 
 file(GLOB_RECURSE ALL_VIAM_HDRS "src/*.h")
 file(GLOB_RECURSE ALL_VIAM_SRCS "src/*.cc")
+file(GLOB_RECURSE ALL_VIAM_C_SRCS "src/*.c")
```

Comment:
This makes it so that we can compile bstring (a C library for safer strings in C): https://bstring.sourceforge.net/

Reply:
The 5 files below are all bstring files.